src/vue-apollo.js

```diff
@@ -167,58 +163,6 @@ const authMiddleware = setContext(async (_, { headers }) => {
     }
   }
 
-  if (store.getters['auth/legacy']) {
-    if (_.operationName == 'RefreshToken') {
-      // The refresh route requires the refresh token to be
-      // sent as the authorization header, with the
-      // access token as the body
-      return {
-        headers: {
-          ...headers,
-          authorization: `Bearer ${store.getters['auth/refreshToken']}`,
-          'authorization-expiry': store.getters['auth/refreshTokenExpiry']
-        }
-      }
-    }
-
-    // The login route requires that no authorization header be sent
-    if (_.operationName == 'LogIn') {
-      return {
-        headers: {
-          ...headers
-        }
-      }
-    }
-
-    const authRefreshRequired =
-      store.getters['auth/authorizationToken'] &&
-      aboutToExpire(store.getters['auth/authorizationTokenExpiry'])
-
-    const validRefreshToken =
-      store.getters['auth/refreshToken'] &&
-      notExpired(store.getters['auth/refreshTokenExpiry'])
-
-    const isAuthenticatedUser =
-      store.getters['auth/idToken'] &&
-      notExpired(store.getters['auth/idTokenExpiry'])
-
-    const middleOfRefresh =
-      store.getters['auth/isRefreshingAuthorization'] ||
-      store.getters['auth/isAuthorizingUser'] ||
-      store.getters['auth/isLoggingInUser']
-
-    if (authRefreshRequired && !middleOfRefresh) {
-      defaultApolloClient.cache.reset()
-      if (validRefreshToken) {
-        await store.dispatch('auth/refreshAuthorization')
-      } else if (isAuthenticatedUser) {
-        await store.dispatch('auth/authorize')
-      } else {
-        await store.dispatch('auth/login')
-      }
-    }
-  }
-
```

Comment:
All of this has been removed in favor of letting the auth layer handle everything related to token refreshing, with the terminal link handling in-flight requests that fail token checks due to expiration.